galaxykit/groups.py

```diff
@@ -2,38 +2,38 @@
 import json
 
 
-def find_group(galaxy_root, headers, group_name):
+def find_group(client, group_name):
     """
     Returns the data of the group with group_name
     """
-    all_groups = requests.get(f"{galaxy_root}_ui/v1/groups/", headers=headers).json()
-    match_group = [x for x in all_groups["data"] if x["name"] == group_name]
-    return match_group
+    groups_url = f"_ui/v1/groups/?name={group_name}"
+    return client.get(groups_url)
+
+def get_group_id(client, group_name):
+    """
+    Returns the id for a given username
```

Comment:
nit: the comment here is wrong, it should be 'for the given _group_ name'

Reply:
fixed